src/Maths/Silk.NET.Intrinsics/Avx/AvxRegister.Float.cs

```diff
@@ -0,0 +1,330 @@
+// This file is part of Silk.NET.
+// 
+// You may modify and distribute Silk.NET under the terms
+// of the MIT license. See the LICENSE file for details.
+
+namespace Silk.NET.Intrinsics.Avx
+{
+    public partial struct AvxRegister : IRegister<float>
+    {
+        public WorkUnit<float> LengthSquared(WorkUnit<float> vector)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public WorkUnit<float> Length(WorkUnit<float> vector)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public WorkUnit<float> Normalize2(WorkUnit<float> vector)
```

Comment:
I'm not sure `Normalize2` is a "clear" name. I understand what it means with context, but it isn't immediately obvious.

Reply:
XML docs will cover that when we add them post-development.

Reply:
You might also consider renaming to `NormalizeVector2` which would disambiguate and not force users to rely on docs.